classes/class-uagb-init-blocks.php

```diff
@@ -941,6 +952,11 @@ public function add_gbs_class( $block_content, $block ) {
 	 * @since x.x.x
```

Comment:
https://docs.google.com/document/d/1pcSs5OrvHrJG5DDKPeInU5H_JGZz_sa2ylD1LGSVoB0/edit#heading=h.yvfss3t6ia8p

```suggestion
	 * @since x.x.x
	 * @return void
```

Reply:
Done